Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb

```diff
@@ -4,7 +4,10 @@
   "metadata": {
```

Comment:
<p>I don't like how the moving text looks, it makes it look very unproffesional. If we can remove this . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Sure we can...